badges/badges.css

```diff
@@ -0,0 +1,65 @@
+@import "../avatar/avatar.css";
+
+*{
+    --online-color: #108a2c;
```

Comment:
Just a suggestion, for the colors used on badges to be bright especially those used for status depiction.